pip/requirements.txt

```diff
@@ -79,6 +79,7 @@ GitPython==3.1.0 ; python_version>'3.0'
 google-common==0.0.1
 google-pasta==0.2.0
 grpcio==1.27.2
+grpcio-tools==1.27.2
```

Comment:
should we update this version (and the above grpcio version) to be consistent with 1.28.1?